src/Controller/BatchProcessingController.py

```diff
@@ -320,6 +325,39 @@ def batch_dvh2csv_handler(self, interrupt_flag,
         self.batch_summary[patient]['dvh2csv'] = reason
         progress_callback.emit(("Completed DVH2CSV", 100))
 
+    def batch_pyrad2csv_handler(self, interrupt_flag,
+                                progress_callback, patient):
+        """
+        Handles creating, starting, and processing the results of batch
+        Pyrad2CSV.
+        :param interrupt_flag: A threading.Event() object that tells the
+                               function to stop loading.
+        :param progress_callback: A signal that receives the current
+                                  progress of the loading.
+        :param patient: The patient to perform this process on.
+        """
+        # Get current files
+        cur_patient_files = \
+            BatchProcessingController.get_patient_files(patient)
+        process = BatchProcessPyRad2CSV(progress_callback,
+                                        interrupt_flag,
+                                        cur_patient_files,
+                                        self.pyrad_output_path)
+        process.set_filename('PyRadiomics_' + self.timestamp + '.csv')
+        success = process.start()
+
+        # Set summary message
+        if success:
+            reason = "SUCCESS"
```

Comment:
If you can make summary messages enum or constants, and import them when you want to use that would be cleaner.